AUTHORS

```diff
@@ -26,3 +26,4 @@ Rafael Audibert
 Steffen Grundke
 WL
 wolffshots
+Divyateja04
```

Comment:
I think this should be sorted.

Reply:
means I shouldn't have added my name?

Reply:
like I saw that your bot was asking so I added before itself, so I thought I'll add

Reply:
No, you can absolutely add your name. But the list should be sorted alphabetically.

Reply:
oh like that...yeah I'll do that

Reply:
yeah done, can you check the PR?